package.json

```diff
@@ -121,7 +121,10 @@
                 },
                 "args": [
                     "/debugAdapter",
-                    "/logLevel:Verbose"
+                    "/logLevel:Verbose",
+                    "/hostName:\"Visual Studio Code Host\"",
+                    "/hostProfileId:Microsoft.VSCode",
+                    "/hostVersion:0.5.0"
```

Comment:
Which "host" version is this?  VSCode or PSES?  If PSES, can't we read the version number from an assembly attribute via refection?